specification/machinelearning/resource-manager/Microsoft.MachineLearning/stable/2017-01-01/webservices.json

```diff
@@ -1219,6 +1230,66 @@
           }
         }
       }
+    },
+    "OperationEntityListResult": {
+      "description": "The list of REST API operations.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of operations.",
```

Comment:
please mark properties of these 3 types as readonly since they're only (ever) returned, not sent

Reply:
Done